packages/nextjs/app/builders/0x84fF709680E9378734Bae1D3B55C6568d21671c3/page.tsx

```diff
@@ -0,0 +1,31 @@
+//import Link from "next/link";
+import { BellIcon, BriefcaseIcon } from "@heroicons/react/24/outline";
+import { NextPage } from "next";
+
+
+const KhayeBeth: NextPage = () => {
+    return (
+        <div className="flex flex-col items-center">
```

Comment:
Maybe add also add `container mx-auto p-8` className's to this so that it's centered on large screens and also text has some space above header ?

Reply:
I just realized i push the repo that I had not installed yarn and all